python/llm/dev/benchmark/all-in-one/run.py

```diff
@@ -258,7 +263,63 @@ def run_optimize_model(repo_id,
             result[in_out] = []
             for i in range(num_trials + warm_up):
                 st = time.perf_counter()
-                output_ids = model.generate(input_ids, do_sample=False, max_new_tokens=out_len)
+                output_ids = model.generate(input_ids, do_sample=False, max_new_tokens=out_len, use_cache=True)
+                end = time.perf_counter()
+                print("model generate cost: " + str(end - st))
+                output = tokenizer.batch_decode(output_ids)
+                print(output[0])
+                if i >= warm_up:
+                    result[in_out].append([model.first_cost, model.rest_cost_mean, model.encoder_time])
+    return result
+
+
+def run_original_model_cpu(repo_id,
+                       local_model_hub,
+                       in_out_pairs,
+                       warm_up,
+                       num_trials):
+    from transformers import AutoTokenizer, LlamaTokenizer, AutoModel, AutoModelForCausalLM
+
+    model_path = get_model_path(repo_id, local_model_hub)
+    st = time.perf_counter()
+    if repo_id in ['THUDM/chatglm-6b', 'THUDM/chatglm2-6b']:
+        model = AutoModel.from_pretrained(model_path, trust_remote_code=True, torch_dtype='auto', low_cpu_mem_usage=True).float()
+        tokenizer = AutoTokenizer.from_pretrained(model_path, trust_remote_code=True)
+    elif repo_id in ['meta-llama/Llama-2-70b-chat-hf']:
+        model = AutoModelForCausalLM.from_pretrained(model_path,
+                                                     trust_remote_code=True, optimize_model=False, low_cpu_mem_usage=True)
```

Comment:
`optimize_model` parameter of `from_pretrained` is for our `bigdl.llm.transformers.AutoModelForCausalLM`.

Reply:
Got it, I'll remove line 288-291.